site_scons/components/__init__.py

```diff
@@ -134,6 +134,7 @@ def define_mercury(reqs):
                  '--prefix=$OFI_PREFIX',
                  '--disable-efa',
                  '--disable-psm3',
+                 '--disable-opx',
```

Comment:
Is this being done in the RPM build also?

Reply:
We do have "--disable-opx" in libfabric.spec of libfabric PR. 
%global configopts --enable-sockets --enable-verbs --enable-usnic --disable-static --disable-efa --without-gdrcopy --disable-opx --enable-tcp